sdk/basyx/object_store.py

```diff
@@ -87,6 +88,14 @@ def __init__(self, objects: Iterable[_IdentifiableType] = ()) -> None:
         for x in objects:
             self.add(x)
 
+    def descend(self) -> Iterator["Class"]:
```

Comment:
Can you please add a docstring what this does and how to use it?

Same for all other methods, some seem to not be documented.

Reply:
I have added docstrings for all methods of the objectstore.